sdk/core/azure-core/inc/azure/core/internal/json.hpp

```diff
@@ -2161,14 +2161,14 @@ _az_JSON_HEDLEY_DIAGNOSTIC_PUSH
     - @ref out_of_range for exceptions indicating access out of the defined range
     - @ref other_error for exceptions indicating other library errors
 
-    @internal
+    \@internal
```

Comment:
Maybe it's none of my business. But you've made a lot of changes to a third-party dependency file. What if in the future we decide to upgrade to a new version?

Reply:
It won't be a first change to this file - we already had to rename macros (tens of them) - we prefixed them with `_az_`. And the file itself is artificially crafted from multiple files - cpps an headers. So, if we pull a new version, we'll have to go through the same process - putting multiple files into one, removing include guards, pulling the list of macros, renaming macros, putting `#undef`s for them at the end, removing `and`s, `or`s, and `not`s (compiler will remind with failure), fixing the doxygen tags (compiler will remind with failure. IMO, this is laborious, but pretty monotonic.

Reply:
And absolutely, thank you for the question, this is a good question, and a valid one, please always ask if you see something, I really appreciate it, and if you have concerns please continue conversation, I value all the feedback, I don't consider anything in our codebase as "not your business".